### PR TITLE
[ADD] postgres_16_support

### PR DESCRIPTION
**Pull request by pleirb**
I took the courage to create this PR based on the public contribution made by @oskardotglobal 
I honestly don't understand much what the project is about but I have noticed that the ci.yaml file probably needs a review by the @Tecnativa team

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -1,74 +1,77 @@
...
     strategy:
       fail-fast: false
       matrix:
         # Test modern Odoo versions with latest Postgres version
         pg_version:
+          - "16"
           - "15"
-          - "14"
           - "13"
-          - "12"
```
Comment (pedrobaeza) [inline .github/workflows/ci.yaml:40]: Why removing these versions?

**Review by pedrobaeza — COMMENTED** [review]
Thanks for the contribution

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (pleirb) [reply .github/workflows/ci.yaml:40]: I don't know, it was like this in the original code provided by @oskardotglobal. Any reason for it?

Comment (oskardotglobal) [reply .github/workflows/ci.yaml:40]: I didn't have any need for them personally. Literally just that

**Comment by oskardotglobal** [comment]
Also, might wanna re-add the unit tests & pre-commit steps

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (pedrobaeza) [reply .github/workflows/ci.yaml:40]: OK, then better to keep compatibility while working if there's no other reason to remove them

**Comment by pleirb** [comment]
> @pleirb if the work is from @oskardotglobal, you should preserve hist attribution in the commit.

Yes, I would prefer that the PR be done by @oskardotglobal
Honestly, I am not qualified for this project at this time, my intention is to promote and update the tool.

**Comment by celm1990** [comment]
@pedrobaeza I am interested in enabling PG 16 here and in the doodba-copier-template. What is the status of this PR?

**Comment by oskardotglobal** [comment]
@celm1990 
I will create a PR for the new workflow some time, I still have to integrate the current testing & pre-commit checks

**Comment by celm1990** [comment]
> @celm1990 I will create a PR for the new workflow some time, I still have to integrate the current testing & pre-commit checks

Ok, if we can join efforts, please let me know. I am available to collaborate.

**Comment by pleirb** [comment]
> .celm1990 Crearé una PR para el nuevo flujo de trabajo en algún momento, todavía tengo que integrar las pruebas actuales y cheques previos al compromiso

I think we would all be very happy for this PR

**Comment by rrebollo** [comment]
@oskardotglobal @celm1990 @pleirb is there anything I can do to help getting this done? odoo.sh is already supporting PG 16, I'm unable to deploy local copies from my projects using doodba.

**Comment by pleirb** [comment]
> @celm1990@pleirbEspárquea de la policía hago para ayudar a hacer esto? odoo.sh ya está apoyando PG 16, no puedo desplegar copias locales de mis proyectos usando doodba.

At the moment what I am doing is using this db image in common.yaml:
`ghcr.io/cmwedding-it/postgres-autoconf:16-alpine`

**Comment by bosd** [comment]
Can you explain some of the other changes?

**Comment by pleirb** [comment]
> Can you explain some of the other changes?

Actually, this PR is a copy of what @oskardotglobal proposes, that at some point when I can make the official integration to this repository. There are people who may be willing to collaborate like: @celm1990 and @rrebollo
I can try to do something and @pedrobaeza can surely indicate the best way to proceed so that support is made officially
If you want to contribute too, I'm sure you'll get support from the moderators.

**Comment by oskardotglobal** [comment]
Alright, I've gone ahead and done the PR myself. See #18.

**Comment by pleirb** [comment]
> Alright, I've gone ahead and done the PR myself. See #18.

Thank you !

**Pull request closed without merging**